server/lib/translator.js

```diff
@@ -0,0 +1,32 @@
+// Allows to access env variables
+const dotenv = require("dotenv").config();
+
+// // Add in local .env file
+// // Google project_id for google translate api
+//projectId=""
+//// Imports the Google Cloud client library
+//key=""
+
+async function quickStart(target, text) {
+  const projectId = process.env.projectId;
+
+  const key = process.env.key;
+  const { Translate } = require("@google-cloud/translate");
```

Comment:
convert this to an import statement and put it on top of the file

Reply:
According to my observation import statements are not my team's practice on the server side.

Reply:
I see, that's fair